internal/venice-common/src/main/java/com/linkedin/venice/ConfigKeys.java

```diff
@@ -848,6 +848,8 @@ private ConfigKeys() {
    */
   public static final String ROUTER_MAX_OUTGOING_CONNECTION = "router.max.outgoing.connection";
 
+  public static final String ROUTER_PER_STORE_THROTTLER_ENABLED = "router.per.store.throttler.enabled";
```

Comment:
Can you add some javadoc for the newly added config options?

Reply:
I think we should rename it to be 
`router.per.storage.node.throttler.enabled`
There is still a per store throttler, but in Router level.

Reply:
its `per.store` only, not per `storage.node`?

Reply:
Currently in Router, the throttler is always per router since Router is not using distribute quota.
Today, there are two throttlers:
1. per store throttler.
2. per store per storage node throttler.
And the config option is for #2, right?
If yes, we should make it more specific, maybe call it `router.per.store.per.storage.node.throttler.enabled`